tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -14,6 +14,13 @@ namespace clang {
 namespace spirv {
 
 uint32_t TypeTranslator::translateType(QualType type) {
+  {
```

Comment:
It was there to give `canonicalType` a much smaller scope. But I'm fine to remove it. Will do.